<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>135</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="boton_buscar_archivo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="texto_archivo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>137</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_archivo">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>42</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Archivo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fileName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>401</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="openFile">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>300</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Abrir Archivo</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="fileContent">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabla_tokens">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Token</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cantidad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Linea</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_errores">
         <property name="text">
          <string>Errores Lexicos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="texto_errores">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="fileValues">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>255</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>33</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
